src/cdk/observers/observe-content.spec.ts

```diff
@@ -120,6 +120,58 @@ describe('Observe content', () => {
   });
 });
 
+describe('ContentObserver injectable', () => {
+  describe('basic usage', () => {
+    let callbacks: Function[];
+    let invokeCallbacks = (args?: any) => callbacks.forEach(callback => callback(args));
+
+    beforeEach(fakeAsync(() => {
+      callbacks = [];
+
+      TestBed.configureTestingModule({
+        imports: [ObserversModule],
+        declarations: [UnobservedComponentWithTextContent],
+        providers: [{
+          provide: MutationObserverFactory,
+          useValue: {
+            create: function(callback: Function) {
+              callbacks.push(callback);
+
+              return {
+                observe: () => {},
+                disconnect: () => {}
+              };
+            }
+          }
+        }]
+      });
+
+      TestBed.compileComponents();
+    }));
+
+    it('should trigger the callback when the content of the element changes',
+        fakeAsync(inject([ContentObserver], (co: ContentObserver) => {
+          let fixture = TestBed.createComponent(UnobservedComponentWithTextContent);
+          fixture.detectChanges();
+
+          const spy = jasmine.createSpy('content observer');
+
+          const subscription = co.observe(fixture.componentInstance.contentEl.nativeElement)
+              .subscribe(() => spy());
+
+          expect(spy).not.toHaveBeenCalled();
+
+          fixture.componentInstance.text = 'text';
+          fixture.detectChanges();
+          invokeCallbacks();
+
+          expect(spy).toHaveBeenCalled();
+
+          subscription.unsubscribe();
+        })));
+  });
```

Comment:
* Should add a test for observing the same element twice, unobserving once and ensuring you still get events. 
* Should add a test for debouncing if it's part of the API